.github/CODEOWNERS

```diff
@@ -1 +0,0 @@
-* @mapbox/navigation-android
```

Comment:
Are we really removing this? The script that notifies people to give a review for pull requests seems like it should not be the same as this restriction.